frontend/Panel.js

```diff
@@ -310,6 +324,12 @@ var styles = {
     padding: 30,
     flex: 1,
   },
+  errorMessage: {
+    color: '#f38b00',
```

Comment:
I don't think we use any color in other places. Let's keep the style consistent?